src/middlewares/hash.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable require-jsdoc */
```

Comment:
This file is more of a helper than a middle-ware
Because it is not used as a middle-ware in any of the routes of the project

My suggestion is to move it to the helper function